core/module/ancestors_spec.rb

```diff
@@ -38,5 +38,33 @@ class << ModuleSpecs::Child; self; end.ancestors.should include(ModuleSpecs::Int
                                       BasicObject)
 
     end
+
+    describe 'for a standalone module' do
+      it 'does not include Class' do
+        SMod = ModuleSpecs.singleton_class
+        SMod.ancestors.should_not include(Class)
```

Comment:
Mmmh, because it's done like that just above I guess. Can you switch to local variables and I'll update the spec above?